### DISCUSSION TITLE
Some tags mess with the heading 1

### DISCUSSION THREAD

**Discussion by NFanoe** [squidfunk/mkdocs-material#7575] [Q&A] [answered]
I'm looking into using tags, and I've defined some like this:

```json
---
tags:
  - Overview
  - Solution
---
# XCorp solution overview
...
```
On my tags page, the heading **Overview** shows up correctly, but the link to the page above is titled "Overview". 

Other tagged pages are linked correctly to the first h1, for instance a page called "Integrations using DataReplicator", which is tagged in exactly the same way as the one mentioned above.

Is "Overview" somehow a magic word?

**Comment by alexvoss** [comment]
> link to the page above is titled "Overview".

You mean it is "Overview" instead of "XCorp solution overview"? I cannot immediately see why this would be the case.

You could help us investigate the problem by creating a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/).

**Reply by NFanoe** [reply]
Yes, it is "Overview" rather than "XCorp solution overview". I will test in a minimal configuration and let you know!

**Reply by NFanoe** [reply]
Created virtual environment with new mkdocs material. This is my mkdocs.yml file:

```yaml
site_name: My Docs

theme:
  name: material

plugins:
  - tags:
    tags_file: tags.md

markdown_extensions:
  - attr_list
```

When running `mkdocs build`, I get: 
`ERROR   -  Config value 'plugins': Invalid Plugins configuration`

What gives? Doesn't look wrong to me?

**Reply by alexvoss** [reply]
you need to indent the `tags_file` part relative to `tags` to make the YAML valid

**Reply by squidfunk** [reply]
My guess is that you're missing a newline between the front matter and your headline. Try this:

``` md
---
tags:
  - Overview
  - Solution
---

# XCorp solution overview
...
```

**Reply by NFanoe** [reply]
OMG missing two spaces before tags_file ...

**Reply by NFanoe** [reply]
And of course it works in the minimal setup ... dammit

**Reply by NFanoe** [reply]
@squidfunk Spaces don't help :/

**Reply by alexvoss** [reply]
I also had a look and at least in my mini-project, there did not seem to be a need for space between the header and the first heading. There must be something else going on.

**Reply by NFanoe** [reply]
Ah. I know why now. Not really happy, but I know why ... The nav object in mkdocs has named the pages "Overview". I suppose that takes precedence over the `# Document title`. Any way around that? 
The "Overview" title makes sense in the nav, because you get the context: Solution > Overview, and Project > Overview. But in the tags list, this context does not exist.

**Reply by NFanoe** [reply]
Such as overriding the page title directly in the `tags` tag, like this:
``` yaml
tags:
  - Overview{Xcorp solution overview}
```
-or
``` yaml
tags:
  - Overview{title}
```

**Reply by NFanoe** [reply, marked as answer]
Ha! ChatGPT to the rescue: Add the title in the metadata as `title: Xcorp solution overview`. That works! :)

**Reply by alexvoss** [reply]
Yes, that explains it. Unfortunately, there is a lack of clarity about the origins and uses of titles that we inherit from MkDocs (if you are interested, have a look at [the links in this post](https://github.com/squidfunk/mkdocs-material/discussions/7310#discussioncomment-9925400). 

At the same time, it is a good example of why it is always a good idea to produce a [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/) as this would have highlighted where the problem lies.
